utils/steps.spec.jsx

```diff
@@ -18,17 +19,33 @@ describe('FormWizard', () => {
     { id: 'second-step', title: 'Second Step', components: ['foo', 'bar'] },
     { id: 'third-step', title: 'Third Step', components: ['foobar'] },
   ];
+  const customStep = { confirmation: PersonConfirmation };
+  const emptyStep = { confirmation: null };
```

Comment:
If we specify `customStep?.confirmation` in the `createSteps` we don't need to pass the `emptyStep`.